includes/traits/llms-trait-award-templates-post-list-table.php

```diff
@@ -60,7 +60,7 @@ public function add_post_actions( $actions, $post ) {
 			add_query_arg(
 				array(
 					LLMS_Admin_Post_Table_Awards::TEMPLATE_FILTER_QUERY_VAR => $post->ID,
-					'post_type'                                             => $award_post_type,
+					'post_type' => $award_post_type,
```

Comment:
stupid CS rule... this is buggy.

Reply:
I'm not going to try to rewrite the PHPCS rules to fix this. Thanks for your feedback.

Reply:
I'm not rewriting the linter or beautifier either. It's all coming frmo "upstream" in either WPCS or PHPCS itself. This barely matters.

Reply:
yeah sure, it was just a comment :P